docs/building.md

```diff
@@ -30,7 +30,13 @@ https://github.com/bazelbuild/bazelisk/releases
 
 ## Configure the build
 
-Note: the `configure` script requires Python 3.
+Use the `configure` script to configure your built. Note: the `configure` script
+requires Python 3.
+
+By default, running the `configure` script will:
+* Create and/or update your conda environment
+* Configure pre-commit hooks for development purposes
+* Configure bazelisk based on the device you're using
```

Comment:
What does 'based on the device you're using mean?

Are you talking about the --config line?

I'd say:

* Configure bazel based on your host OS

Reply:
Yes, that's what I meant. I've changed the wording on this.